ext/standard/uniqid.c

```diff
@@ -72,12 +74,17 @@ PHP_FUNCTION(uniqid)
 	gettimeofday((struct timeval *) &tv, (struct timezone *) NULL);
 	sec = (int) tv.tv_sec;
 	usec = (int) (tv.tv_usec % 0x100000);
+	php_random_bytes(bytes, UNIQID_RAND_CHARS*2, 1);
+	for (i = 0; i < UNIQID_RAND_CHARS; i++) {
+		bytes[i] = convtab[bytes[i] & mask];
+	}
+	bytes[UNIQID_RAND_CHARS] = '\0';
 
 	/* The max value usec can have is 0xF423F, so we use only five hex
 	 * digits for usecs.
 	 */
 	if (more_entropy) {
-		uniqid = strpprintf(0, "%s%08x%05x%.8F", prefix, sec, usec, php_combined_lcg() * 10);
+		uniqid = strpprintf(0, "%s%08x%05x%s", prefix, sec, usec, bytes);
```

Comment:
@Ocramius 
Why? It could break code, but almost all code does not care about return value from uniqid().